friendly/src/main/webapp/script.js

```diff
@@ -47,8 +49,11 @@ function getUserGroups() {
         else {
             groupData.innerHTML = ('<h4>Your groups: </h4>');
             groupDropdown = document.createElement('select');
+            groupDropdown.setAttribute("id", "groupId");
+            groupDropdown.setAttribute("onchange","getRecommendationContainer()"); //this is not working
```

Comment:
i think `groupDropdown.onchange = getRecommendationContainer()` should work

Reply:
Oops. groupDropdown.setAttribute() works, I forget to remove the comment